app/index.js

```diff
@@ -247,7 +248,13 @@ module.exports = generators.Base.extend({
       }
     }
   },
-  install: function () {
-    if(!this.options['skip-install']) this.installDependencies();
+  install: function() {
+    if (!this.options['skip-install']) {
+      return this.installDependencies(function() {
+        return spawn('npm', ['run', 'test:coverage'], {
```

Comment:
@petecoop i made it run the tests after install of new apps, and added the scripts to be run on jenkins so you have more confidence in new builds and releases.